source/example.config.ts

```diff
@@ -11,9 +11,14 @@ export const db = {
 };
 export const authorization = {
 	/**  lifetime of the token in the browser in ms */
-	tokenLifeBrowser: 100_000,
+	tokenLifeBrowser: 172_800_000,
```

Comment:
Is this not too long?

Reply:
It's 2 days, I think it's long enough to be reasonably secure (namely, not have random tokens floating around somewhere that an attacker can yoink) while also not inconveniencing the user. Ideally the token should also be refreshed occasionally, maybe with every request or sth.